logger.js

```diff
@@ -1,13 +1,8 @@
 const AWS = require('aws-sdk');
 
 class Logger {
-    constructor({stage, stack, app, roleArn, streamName, sendLogsToKinesis = true}) {
-        this.stage = stage;
-        this.stack = stack;
-        this.app = app;
-        this.roleArn = roleArn;
-        this.streamName = streamName;
-        this.sendLogsToKinesis = sendLogsToKinesis;
+    constructor(options) {
+        Object.assign(this, { sendLogsToKinesis: true }, options)
```

Comment:
Flow would enforce it if you had flow on the project you're importing so probably not overly useful. If we *did* have flow, then you'd get the signature though so that would be useful. I do quite like that required options pattern though, it'd be nice to extract that into a module.

This is an interesting pattern too (though doesn't help with the `Object.assign` approach): https://github.com/MarkTiedemann/throw-if-missing